src/reference/02-DetailTopics/05-Plugins-and-Best-Practices/05-Testing-sbt-plugins.md

```diff
@@ -35,7 +35,7 @@ Add scripted-plugin to your plugin build. `project/scripted.sbt`:
 libraryDependencies += { "org.scala-sbt" % "scripted-plugin" % sbtVersion.value }
 ```
 
-Then add the following settings to `scripted.sbt`:
+Then add the following settings to `build.sbt`:
```

Comment:
adding it to scripted.sbt as is results in an `error: not found: value ScriptedPlugin` for me